tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -431,7 +431,7 @@ export const openAndGetContentOfDocument = async ({
   editorToOpen: string
 }): Promise<string> => {
   await page.waitForLoadState()
-  await page.waitForURL('**/external/**')
+  await page.waitForURL('**/external**')
```

Comment:
Would this:

```suggestion
  await page.waitForURL('**/external-**')
```

mean that the redirect page is not being tracked and the test can only succeed when we are successfully being redirect to `external-onlyoffice`, `external-collabora`, whatsoever? Then that would be a good thing.